javascript-data-model/order-history.js

```diff
@@ -0,0 +1,61 @@
+var orderHistory = [
+  {
+    title: 'Javascript for impatient programmers',
+    type: 'book',
+    author: 'Dr. Axel Rauschmayer',
+    totalCost: 34.00,
+    itemCost: 31.55,
+    orderPlaced: 'August 4, 2020',
+    orderDelivered: 'August 8, 2020',
+    returnWindowClosed: 'September 7, 2020',
+    orderNumber: '114-3941689-8772232'
```

Comment:
This data seems to relate to the order itself. I think the main object stored in each index of the `orderHistory` variable should describe the order, and then inside of that object, you could have a property named `items` that includes the information about each item ordered.